### PR TITLE
Improve(ots_table): Improve its resource and datasource use common method.

### PR DESCRIPTION
**Pull request by Lexsss**

### CODE REVIEW

File: alicloud/service_alicloud_ots.go [hunk 1]
```diff
@@ -33,90 +33,95 @@ func (s *OtsService) getPrimaryKeyType(primaryKeyType string) tablestore.Primary
...
 		}
+		addDebug("ListTable", raw)
 		return nil
 	})
-
+	if err != nil {
+		if strings.HasPrefix(err.Error(), OTSObjectNotExist) {
+			return table, WrapErrorf(err, NotFoundMsg, AliyunTablestoreGoSdk)
+		}
+		return nil, WrapErrorf(err, DataDefaultErrorMsg, instanceName, "ListTable", AliyunTablestoreGoSdk)
```
Comment (YY-tomorrow) [inline alicloud/service_alicloud_ots.go]: DataDefaultErrorMsg change to DefaultErrorMsg

File: alicloud/service_alicloud_ots.go [hunk 2]
```diff
@@ -33,90 +33,95 @@ func (s *OtsService) getPrimaryKeyType(primaryKeyType string) tablestore.Primary
...
+	}
+	table, _ = raw.(*tablestore.ListTableResponse)
+	if table == nil {
+		return table, WrapErrorf(Error(GetNotFoundMessage("OtsTable", instanceName)), NotFoundMsg, ProviderERROR)
+	}
 	return
 }
 
 func (s *OtsService) DescribeOtsTable(instanceName, tableName string) (table *tablestore.DescribeTableResponse, err error) {
```
Comment (YY-tomorrow) [inline alicloud/service_alicloud_ots.go]: please use id

File: alicloud/resource_alicloud_ots_table.go [hunk 3]
```diff
@@ -53,6 +53,7 @@ func resourceAlicloudOtsTable() *schema.Resource {
 					},
 				},
 				MaxItems: 4,
+				ForceNew: true,
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_ots_table.go]: update the document

File: alicloud/service_alicloud_ots.go (hunk 1 above)
Comment (Lexsss) [reply alicloud/service_alicloud_ots.go]: called by datasource only

**Pull request merged**